src/stickyfill.js

```diff
@@ -7,8 +7,11 @@
  */
 let seppuku = false;
 
+// Check for existence of window in case this is being imported on the server in an SSR app
+const isWindowDefined = typeof window !== 'undefined';
```

Comment:
Never used SSR before, os I’m not sure what’s the best way to make this work on the server side. Can we just use the global object instead of `window`? Will it make any sense? Like, doing `const global = this` at the beginning of the script and then using it instead of `window` throughout the script?

Or maybe we should just set `seppuku` flag on the server and prevent the polyfill from running? (And since we also intoduce `forceSticky` method, should we ignore it on the server?)

Reply:
Basically in SSR you have the same code running in both the client and the server. The reason this is breaking on the server is because this code executes immediately on the server upon importing. So any immediately-executed code that references `window` will break because `window` is `undefined`.

I think adding this `isWindowDefined` check is the easiest way to handle this problem without introducing a breaking change. Two other options (both requiring a change in the API) would be:

1. Exporting the `init` function as a property on `Stickyfill` rather than having it be part of the main exported code, so that consumers of the package are responsible for manually calling `init` in order for any client side code dependent on `window` to be run
2. Wrapping the whole module in a function, like this (now out-of-date) fork of your project: https://github.com/webmodules/stickyfill/blob/master/index.js

I'll defer to you on how you'd like to handle this! For now just having the `isWindowDefined` check should work, since either of the two options above would be considered a breaking change. Thoughts?